### PR TITLE
fix tagging for api sources (required reviewers)

### PR DESCRIPTION
**Pull request by erohmensing**
## What
* Handle tagging for API sources

## How
* critical connectors auto-tagged via CODEOWNERS
* other api sources not auto-tagged at the moment, we can re-evaluate if we want it back

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by erohmensing** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37728?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#37728** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37728?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#37716** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37716?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#37724** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37724?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @erohmensing and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -51,3 +51,10 @@
 /airbyte-integrations/connectors/destination-s3/ @airbytehq/destinations
 /airbyte-integrations/connectors/destination-snowflake/ @airbytehq/destinations
 /airbyte-integrations/connectors/destination-redshift/ @airbytehq/destinations
+
+# Python critical connectors
+/airbyte-integrations/connectors/source-facebook-marketing/ @airbytehq/critical-connectors
+/airbyte-integrations/connectors/source-hubspot/ @airbytehq/critical-connectors
+/airbyte-integrations/connectors/source-salesforce/ @airbytehq/critical-connectors
+/airbyte-integrations/connectors/source-shopify/ @airbytehq/critical-connectors
+/airbyte-integrations/connectors/source-stripe/ @airbytehq/critical-connectors
```
Comment (erohmensing) [inline .github/CODEOWNERS:60]: @maxi297 i think you need to add this repository to the [team's repositories ](https://github.com/orgs/airbytehq/teams/critical-connectors/repositories) for this to resolve

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -51,3 +51,10 @@
 /airbyte-integrations/connectors/destination-s3/ @airbytehq/destinations
 /airbyte-integrations/connectors/destination-snowflake/ @airbytehq/destinations
 /airbyte-integrations/connectors/destination-redshift/ @airbytehq/destinations
+
+# Python critical connectors
+/airbyte-integrations/connectors/source-facebook-marketing/ @airbytehq/critical-connectors
```
Comment (maxi297) [inline .github/CODEOWNERS:56]: I have this error on my side. I'm would like to update the team visibility following [this guide](https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility) but I can't select teams at step 4.

image

File: .github/CODEOWNERS (hunk 1 above)
Comment (maxi297) [reply .github/CODEOWNERS:60]: Done! It seems like it resolved the error for me

**Review by maxi297 — APPROVED** [review]
I'm fine with the code ownership part of the change

**Review by bazarnov — APPROVED** [review]

**Comment by erohmensing** [comment]
### Merge activity

* **May 1, 5:39 PM EDT**: @erohmensing started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/37728).

**Pull request merged**